src/components/Modals/CreateMultipleItemsModal/CreateMultipleItemsModal.tsx

```diff
@@ -0,0 +1,415 @@
+                        )
+                      })}
+                    </Table.Body>
+                  </Table>
+                ) : null}
+                {validFiles.length > 0 ? (
+                  <Table basic="very" compact="very">
+                    {rejectedFiles.length > 0 ? (
+                      <Table.Header>
+                        <Table.Row>
+                          <Table.HeaderCell>{t('create_multiple_items_modal.valid_title')}</Table.HeaderCell>
+                        </Table.Row>
+                      </Table.Header>
+                    ) : null}
+                    <Table.Body>
+                      {validFiles.map(({ fileName }, index) => {
+                        return (
+                          <Table.Row key={index}>
+                            <Table.Cell colSpan="2">{fileName}</Table.Cell>
+                          </Table.Row>
+                        )
+                      })}
+                    </Table.Body>
+                  </Table>
+                ) : null}
+              </div>
+            </div>
+            {rejectedFiles.length > 0 ? (
+              <div className={styles.rejectedFilesInfo}>
+                <InfoIcon className={styles.infoIcon} />
+                {t('create_multiple_items_modal.only_valid_items_info')}
+              </div>
+            ) : null}
+          </div>
+        </Modal.Content>
+        <Modal.Actions>
+          <Button secondary onClick={open}>
+            {t('create_multiple_items_modal.add_more_button')}
+          </Button>
+          <Button primary disabled={validFiles.length === 0} onClick={this.handleFilesUpload}>
+            {t('create_multiple_items_modal.upload_items_button')}
+          </Button>
+        </Modal.Actions>
+      </>
+    )
+  }
+
+  private renderReviewTable = () => {
+    const { onClose } = this.props
+    return (
+      <>
+        <ModalNavigation title={t('create_multiple_items_modal.title')} onClose={onClose} />
+        <Dropzone
+          children={this.renderDropZone}
+          onDropAccepted={this.handleFilesImport}
+          onDropRejected={this.handleRejectedFiles}
+          accept={['.zip']}
+          noClick
+        />
+      </>
+    )
+  }
+
+  private renderImportView = () => {
+    const { onClose } = this.props
+    return (
+      <>
+        <ModalNavigation title={t('create_multiple_items_modal.title')} onClose={onClose} />
+        <Modal.Content>
+          <ItemImport
+            onDropAccepted={this.handleFilesImport}
+            onDropRejected={this.handleRejectedFiles}
+            acceptedExtensions={['.zip']}
+            moreInformation={
+              REACT_APP_WEARABLES_ZIP_INFRA_URL ? (
+                <span>
+                  <T
+                    id="create_multiple_items_modal.import_information"
+                    values={{
+                      link: <a href={REACT_APP_WEARABLES_ZIP_INFRA_URL}>{t('create_multiple_items_modal.import_information_link_label')}</a>
```

Comment:
we could add `rel="noopener noreferrer"` to appease the React Warning gods. Also, maybe a `target="_blank"` ?